app/assets/javascripts/views/story_view.js

```diff
@@ -74,6 +74,20 @@ module.exports = FormView.extend({
 
     // Set up CSS classes for the view
     this.setClassName();
+
+    this.attachinaryOptions = {}
+    this.fetchAttachinaryOptions();
+  },
+
+  fetchAttachinaryOptions: function() {
+    attOptions = new AttachmentOptions({
```

Comment:
It would be good to have a `const` here before the variable name, otherwise, it's gonna create a global.